styles/Navbar.module.css

```diff
@@ -0,0 +1,93 @@
+.header{
+    border-bottom: 1px solid #E2E8F0;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+
+.none {
+    display: none;
+}
```

Comment:
Since this seems like something that could happen globally, could we move this to global.css instead? And then we can just import global to use this.